src/components/data_submission/consent_group/ConsentGroupForm.js

```diff
@@ -12,38 +12,46 @@ export const ConsentGroupForm = (props) => {
     disableDelete,
+    consentGroupsState,
+    studyEditMode,
+    dacs,
   } = props;
 
   const [consentGroup, setConsentGroup] = useState({
-    consentGroupName: '',
+    // this will need to change to studyEditMode ? <consentGroupsState property> : <default>
+    consentGroupName: consentGroupsState[idx].consentGroup.consentGroupName || '',
 
     // primary: one of these needs to be filled
-    generalResearchUse: undefined,
-    hmb: undefined,
-    diseaseSpecificUse: undefined, // string
-    poa: undefined,
-    openAccess: undefined,
-    otherPrimary: undefined, // string
+    // if in edit study mode, give each of these the value of the consent group state at the index
+    // otherwise give them the default value
+    generalResearchUse: consentGroupsState[idx].consentGroup.generalResearchUse || undefined,
+    hmb: consentGroupsState[idx].consentGroup.hmb || undefined,
+    diseaseSpecificUse: consentGroupsState[idx].consentGroup.diseaseSpecificUse || undefined, // string
+    poa: consentGroupsState[idx].consentGroup.poa || undefined,
+    openAccess: consentGroupsState[idx].consentGroup.openAccess || undefined,
+    otherPrimary: consentGroupsState[idx].consentGroup.otherPrimary || undefined, // string
 
     // secondary:
-    nmds: false, // No Methods Development or validation studies
-    gso: false, // genetic studies only
-    pub: false, // publication required
-    col: false, // collaboration required
-    irb: false, // irb approval required
-    gs: null, // string: geographic restriction
-    mor: undefined, // date (string): publication moratorium
-    npu: false, // non profit only
-    otherSecondary: null, // string
+    nmds: consentGroupsState[idx].consentGroup.nmds || false, // No Methods Development or validation studies
+    gso: consentGroupsState[idx].consentGroup.gso || false, // genetic studies only
+    pub: consentGroupsState[idx].consentGroup.pub || false, // publication required
+    col: consentGroupsState[idx].consentGroup.col || false, // collaboration required
+    irb: consentGroupsState[idx].consentGroup.irb || false, // irb approval required
+    gs: consentGroupsState[idx].consentGroup.gs || null, // string: geographic restriction
+    mor: consentGroupsState[idx].consentGroup.mor || undefined, // date (string): publication moratorium
+    npu: consentGroupsState[idx].consentGroup.npu || false, // non profit only
+    otherSecondary: consentGroupsState[idx].consentGroup.otherSecondary || null, // string
 
     // dataLocation is one of:
     // "AnVIL Workspace", "Terra Workspace",
     // "TDR Location", "Not Determined"
-    dataLocation: null,
+    dataLocation: consentGroupsState[idx].consentGroup.dataLocation || null,
 
-    url: '',
-    numberOfParticipants: undefined, // numeric
-    fileTypes: [{}],
+    url: consentGroupsState[idx].consentGroup.url || '',
+    numberOfParticipants: consentGroupsState[idx].consentGroup.numberOfParticipants || undefined, // numeric
```

Comment:
Study ID 8 on dev has participants, but it isn't displayed:
```
        {
          "propertyId": 8992,
          "dataSetId": 1426,
          "propertyName": "File Types",
          "propertyValue": [
            {
              "numberOfParticipants": 3
            }
          ],
          "schemaProperty": "fileTypes",
          "propertyType": "Json"
        }
```
![Screenshot 2023-08-01 at 12 03 54 PM](https://github.com/DataBiosphere/duos-ui/assets/116679/55be724a-db66-438f-8cdf-d20b2d280332)

Reply:
The data here is weird because in this case the number of participants is in the fileTypes properties where in a normal study the number of participants would be it's own property, which is how I extracted it. The property value for fileTypes in this case is corrupt I believe

Reply:
Ah, Ok, got it.